demo/common/assets.js

```diff
@@ -965,6 +965,7 @@ shakaAssets.testAssets = [
       .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
       .addLicenseServer('com.microsoft.playready', 'https://content.uplynk.com/pr')
       .addLicenseServer('com.widevine.alpha', 'https://content.uplynk.com/wv')
+      .setPersistentStateRequired('com.microsoft.playready', true)
```

Comment:
ℹ️ Uplynk PlayReady system sets MediaLicense.Type to `persistent` so we need to enforce the persistent state for those assets